tahowallet/extension
# 🎳 Transmutation: Manifest v3

> Alchemy (錬金術 Renkinjutsu) is, as it is understood in the [Fullmetal Alchemist](https://fma.fandom.com/wiki/Fullmetal_Alchemist) series, the ancient metaphysical science/mystical art of manipulating and altering matter by using natural energy. This act is known as "Transmutation" (錬成 Rensei) and its sequence is usually described as:
> 1. Comprehension - Understanding the inherent structure and properties of the atomic or molecular makeup of a particular material to be transmuted, including the flow and balance of potential and kinetic energy within.
> 2. Deconstruction - Using energy to break down the physical structure of the identified material into a more malleable state so as to be easily reshaped into a new form.
> 3. Reconstruction - Continuing the flow of energy so as to reform the material into a new shape.

https://fma.fandom.com/wiki/Alchemy

# Testing
- load the extension and use normally
- [permission check](https://developer.chrome.com/docs/extensions/mv3/permission_warnings/#view_warnings)
  - clear dist folder
  - checkout main > build > package > load
  - you should see an extension can edit the pages permission modal
  - clear dist folder
  - checkout this branch > build > package (with the same private key) > load 
  - no new permission modal should pop up

## Todo
- [ ] dev time reloader script 
- [ ] usb handling and ledger

Ref #2694

Latest build: [extension-builds-2488](https://github.com/tallycash/extension/suites/9628539326/artifacts/459966499) (as of Fri, 02 Dec 2022 09:50:45 GMT).
